app/assets/javascripts/organisation-autocomplete.js

```diff
@@ -5,16 +5,118 @@ window.GOVUKAdmin.Modules = window.GOVUKAdmin.Modules || {};
 
   Modules.OrganisationAutocomplete = function () {
     this.start = function ($element) {
-      accessibleAutocomplete.enhanceSelectElement({
-        selectElement: $element.find('#organisations')[0],
-        showAllValues: true,
-        autoselect: false,
-        dropdownArrow: dropdownArrow
-      });
-    };
 
-    function dropdownArrow() {
-      return '<span class="caret"></span>';
-    }
+      var autocompleteTemplateHtml = createAutocompleteTemplate();
```

Comment:
I would rename `autocompleteTemplateHtml` to `$template`. Autocomplete is inferred from the module, and HTML is not adding much. I would also add $ as part of the name as this is a `JQuery` object.

Reply:
I don't think we should have the `$` prefix - that denotes a jQuery element, not a raw HTML string.

Reply:
Ok. I would then build a `JQuery` object around it and use the `$`, and only get the `HTML` when needed. I have also scanned the code, and it feels that the variable [is only used once][1], and it basically [undoing this code][2]. 

[1]: https://github.com/alphagov/content-performance-manager/blob/f9e897b0ed61a658d0546cbc3d5861656803ba98/app/assets/javascripts/organisation-autocomplete.js#L64-L64
[2]: https://github.com/alphagov/content-performance-manager/blob/f9e897b0ed61a658d0546cbc3d5861656803ba98/app/assets/javascripts/organisation-autocomplete.js#L18-L18

Reply:
But when we're creating a new HTML element, we don't want to accidentally copy around references to the template object. I think serialising it to a string is the safest way to do this.

Reply:
mmm not sure about it. Let's move the HTML out of the JS, use a template and discuss form there.

Hope this is ok with you @andrewgarner